bigquery/deploy/README.md

```diff
@@ -101,6 +101,31 @@ assume the service account has this name.
   kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
   ```
 
+## Reindex with no downtime
```

Comment:
How about leaving it up to the user? (I also made things a bit less wordy.)


## Reindex on GKE

If you don't have any users and aren't worried about downtime, the easiest way to reindex is to simply rerun `deploy-indexer.sh`.
```
# We recommend deleting the index to start from a clean slate
kubectl exec -it es-data-0 -- curl -XDELETE localhost:9200/DATASET*
bigquery/deploy/deploy-indexer.sh DATASET
# Verily indexing was successful
kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
```
If you have users and don't want them to experience downtime, you can create a second cluster.
```
  kubernetes-elasticsearch-cluster/create-cluster.sh DATASET
  kubernetes-elasticsearch-cluster/deploy-es.sh DATASET
  # Verily elasticsearch was deployed
  kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
  bigquery/deploy/deploy-indexer.sh DATASET
  # Verily indexing was successful
  kubectl exec -it es-data-0 curl localhost:9200/_cat/indices?v
  # Run from data-explorer repo
  deploy/deploy-api.sh DATASET
```
After verifying Data Explorer UI works, don't forget to **delete the old cluster**.

Reply:
Made these changes, thanks!